thoth/digests_fetcher/python/fetch.py

```diff
@@ -38,13 +38,14 @@ def fetch(self, package_name: str, package_version: str) -> dict:
             "Fetching digests for package %r in version %r from %r",
             package_name,
             package_version,
-            self.source.url,
+            self.source.url
         )
 
-        # TODO: implement
+        files = self.source.get_package_hashes(package_name, package_version, True)
+
         return {
             "package_name": package_name,
             "package_version": package_version,
-            "index_url": self.source.index.url,
-            "files": [],
```

Comment:
A lot of time I noticed extra being left at the end, is it a convention?

Reply:
I do it from time to time. It's just my style.